### ISSUE TITLE
Fix color contrast problems

### ISSUE THREAD

**Issue by brenoferreira** [creativecommons/cccatalog-frontend#998]
Lighthouse has found many color contrast problems. Please refer to the [reports](https://drive.google.com/drive/folders/1OeEtEdQhgsXdrlHBBMnARkad_6-xGWno?usp=sharing) to see what the problems are.

Some of the problems found which have low contrast scores are: the tomato color over gray backgrounds, some dark grays over gray, tag labels' gray over white.

**Comment by dos077** [comment]
The <a> color is not defined in this repo, but at creativecommons/vocabulary. The $color-tomato variable is too light on a white black ground, supposedly. But going through Google's own material color palette, it seems like only the deep-orange darken-4 color can satisfy the >4.5 contrast requirement. Not sure how the design process of vocabulary group works, a plain orange text link maybe a big tough to satisfy the accessibility score without looking a bit ugly.

**Comment by Atanda1** [comment]
I'll like to work on this contrast ratio issue

**Comment by dhruvkb** [comment]
@dos077 I am pretty sure that the primary brand colors are immutable and using them is unavoidable. This means running into issues like this is inevitable. That said, here is the only way to rectify this:
- We can redesign the low-contrast parts to be more legible using [the A<sub>n</sub> guide](https://usecontrast.com/guide) and switch to [more legible color combinations](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=0%3A72) in the affected areas.
- We can setup monitoring for accessibility in Vocabulary that will allow us to catch such problems in more components and earlier in their lifecycle.

@Atanda1 this requires some design work. Maybe you can help @panchovm on that.

**Comment by dos077** [comment]
A little more info on the problem. This is where the colors are defined:

https://github.com/creativecommons/vocabulary/blob/master/src/styles/color.scss

Here's Google's material palette:

https://material.io/resources/color/#!/?view.left=0&view.right=0&primary.color=E65100

$color-tomato, which is also the main brand color, is between orange 800-900, neither are sufficiently legible in contrast to white(check accessibility tab). Only primary orange color in this palette that's approved by Google for legibility in contrast to white is deep orange 900, which is pretty dark and doesn't really stand out with black text as the original tomato color.

If accessibility is the primary concern, simply changing $color-tomato to deep oragne 900, rgb(191, 54, 12), will solve the issue, but I am sure this is also a design issue. Maybe I should cross post this to vocabulary's issues?

**Comment by brenoferreira** [comment]
<img width="1402" alt="Screen Shot 2020-06-04 at 13 54 34" src="https://user-images.githubusercontent.com/707019/83787999-00010600-a66b-11ea-9aed-4ff185d20a78.png">
here's an example of tomato over white

**Comment by panchovm** [comment]
Oh. Now I see, we need to work on links. Thanks @brenoferreira

**Comment by Atanda1** [comment]
So what's the conclusion? Are we changing the colour contrast of the link and background

**Comment by panchovm** [comment]
We are changing the tomato color for links. Definitely

**Comment by dhruvkb** [comment]
This seems like a Vocabulary issue. Should I transfer this issue to `creativecommons/vocabulary`?

**Comment by kgodey** [comment]
This is a general issue for all color contrast issues found by Lighthouse, I think it makes sense to leave it here and open new (related) issues in Vocabulary for each of the problems.

**Comment by Atanda1** [comment]
How do I come in now?

**Comment by kgodey** [comment]
@Atanda1 please wait for @panchovm to create new tickets.

**Comment by Atanda1** [comment]
Ok then...I'll look out for that

**Comment by wajeehamushtaq** [comment]
Does anyone working on this issue or may I work on it?

**Comment by annatuma** [comment]
Individual color issues should be addressed in Vocabulary, and updates will be made in parallel to the Design Library.

Leaving to @dhruvkb and @panchovm to create appropriate tickets in Vocabulary (starting with the link color issue), and make changes in Design Library.

We'll create more tickets once the reports are fully reviewed.

For contributors, I recommend looking for issues marked as "help wanted", and keep an eye on the Vocabulary repo for individual tickets there related to this.

**Comment by AyanChoudhary** [comment]
Linking the relevant issues from vocabulary here
https://github.com/creativecommons/vocabulary/issues/454
https://github.com/creativecommons/vocabulary/issues/455

**Comment by Atanda1** [comment]
Thanks! I'll check it out ASAP

**Comment by zackkrida** [comment]
Closed, as all color contrast a11y issues have been resolved in vocabulary.
